project-base/assets/js/frontend/cart/index.js

```diff
@@ -1,3 +1,3 @@
-import './cartBox';
-import './cartRecalculator';
-import './cartBoxItemRemover.js';
+import './CartBox';
+import './CartRecalculator';
+import './CartBoxItemRemover.js';
```

Comment:
Just a question - why is here the `.js` suffix?

Reply:
374c67a886d912b7eaaaeec57089a93414dca4d8